be/src/Project/Web/Seller/Seller.Web/Areas/Products/Repositories/CategoriesRepository.cs

```diff
@@ -231,6 +230,36 @@ public async Task<CategorySchema> GetCategorySchemaAsync(string token, string la
             return default;
         }
 
+        public async Task<IEnumerable<CategorySchema>> GetCategorySchemasAsync(string token, string language, Guid? categoryId)
+        {
+            var apiRequest = new ApiRequest<RequestModelBase>
+            {
+                Language = language,
+                Data = new RequestModelBase(),
+                AccessToken = token,
+                EndpointAddress = $"{this.settings.Value.CatalogUrl}{ApiConstants.Catalog.CategorySchemasApiEndpoint}/{categoryId}"
+            };
+
+            var response = await this.apiClientService.GetAsync<ApiRequest<RequestModelBase>, RequestModelBase, CategorySchemas>(apiRequest);
+
+            if(response.IsSuccessStatusCode && response.Data != null)
+            {
+                return response.Data.Schemas;
+            }
+            
+            if(!response.IsSuccessStatusCode && response.StatusCode == HttpStatusCode.NotFound)
```

Comment:
**suggestion:** I think it is redundant

Reply:
This condition is important because it prevent to showing a exception message on the page.

![image](https://github.com/dawiddworak88/Giuru/assets/112848485/ca49fb27-5959-4114-b099-a1224d67040b)

Reply:
Okay, let's try remove condition from line 255